client/src/pages/Landing/style.css

```diff
@@ -1,2 +1,73 @@
 /* TODO: this entire file is optional, just showing you it can be here */
-/* Use this to add css styles to the Landing React Element */
\ No newline at end of file
+/* Use this to add css styles to the Landing React Element */
+
```

Comment:
On the figma, I used **Poppins** as my font; import it from google fonts or maybe grab something different. I don't love the current font. Maybe add it "globally" in src/App/style.css as the default font to keep the whole site consistent, when possible

Reply:
You don't like helvetica? it's a classic